backend/commons/src/main/resources/db/migration/V1__init_database.sql

```diff
@@ -30,3 +28,66 @@ CREATE TABLE IF NOT EXISTS KNOWN_FINDINGS (
     line_hash VARCHAR(255)
 );
 
+CREATE TABLE IF NOT EXISTS SERVER_FINDINGS (
+    id UUID,
+    connection_id VARCHAR(255) NOT NULL,
+    sonar_project_key VARCHAR(255) NOT NULL,
+    server_key VARCHAR(255) NOT NULL PRIMARY KEY,
+    rule_id VARCHAR(255),
+    rule_key VARCHAR(255) NOT NULL,
+    message VARCHAR(255) NOT NULL,
```

Comment:
I have a doubt with all these capped varchars, for example I wonder is message or project key could be longer than that

Reply:
Changed this one to 10000 limit. We can discuss overall approach later